ext/storage_sources/s3_store/s3_connection.h

```diff
@@ -10,12 +37,18 @@
 #include <string>
 #include <vector>
 
-/*
- * Class to represent an active connection to the AWS S3 endpoint. Allows for interaction with S3
- * client.
- */
+// This class represents an active connection to the AWS S3 endpoint and allows for interaction with
```

Comment:
Can you check if the lines are wrapped as close to 100 character width as possible. Please check at other places too in the code

Reply:
Yes, just checked and these comments are all already wrapped at 100. Will check the other places as well.